server/trpc/router/sendEmail.ts

```diff
@@ -0,0 +1,24 @@
+import toast from "react-hot-toast";
+import { SendEmailSchema } from "../../../schema/comment";
+import sendEmail from "../../../utils/sendEmail";
+import { protectedProcedure, router } from "../trpc";
+
+export const sendEmailRouter = router({
+
+    send:protectedProcedure
+        .input(SendEmailSchema)
+        .mutation(async ({input}) => {
+            const { htmlMessage, subject } = input;
+            try {
+                await sendEmail({
+                    recipient: process.env.ADMIN_EMAIL || '',
+                    htmlMessage,
+                    subject,
+                });
+              } catch (error) {
+                console.log(`Failed to send email: ${error}`)
+                throw new Error();
```

Comment:
```suggestion
                throw new Error(error);
```

Reply:
I had it like that because I didnt want to pass the error to the client. This way the error is thrown, triggering the error toast, and the error message is logged on the server side, not the browser.